hsctl

```diff
@@ -227,6 +231,8 @@ loaddb_hs() {
 
 migrate_hs() {
     echo "MIGRATE:"
+    echo "  - docker exec -u hydro-service hydroshare python manage.py migrate sites"
+    docker exec -u hydro-service hydroshare python manage.py migrate sites
```

Comment:
@hyi - had occasionally run into a site ID issue on builds using a copy of the www db. This seemed to resolve it though it shouldn't technically be necessary... More often than not it is not required.